FeedStoreChallenge/Entities/ManagedFeedImage+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ManagedFeedImage+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Istvan Private on 14.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedFeedImage> {
+		return NSFetchRequest<ManagedFeedImage>(entityName: "ManagedFeedImage")
+	}
+
+	@NSManaged public var id: UUID
+	@NSManaged public var descriptionString: String?
+	@NSManaged public var location: String?
+	@NSManaged public var url: URL
+	@NSManaged public var cache: ManagedCache?
```

Comment:
Since this class should not be public, its functions and methods should be changed also.
```suggestion
	@nonobjc class func fetchRequest() -> NSFetchRequest<ManagedFeedImage> {
		return NSFetchRequest<ManagedFeedImage>(entityName: "ManagedFeedImage")
	}

	@NSManaged var id: UUID
	@NSManaged var descriptionString: String?
	@NSManaged var location: String?
	@NSManaged var url: URL
	@NSManaged var cache: ManagedCache?
```